test/common/README.md

```diff
@@ -116,6 +116,26 @@ Checks whether `IPv6` is supported on this platform.
 
 Checks if there are multiple localhosts available.
 
+### hijackStderr(listener)
+* `listener`
+[&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Normal_objects_and_functions):
```

Comment:
(I know everywhere else uses it this way but) if you are using this link more than one you could use `[MDN-Function][]`, and put
```md
[MDN-Function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Normal_objects_and_functions
```
the the bottom of the file.